src/components/exercise/index.cjsx

```diff
@@ -135,12 +136,15 @@ ExerciseWithScroll = React.createClass
 
   render: ->
     {parts, footer, pinned} = @props
+    classes = classnames('openstax-multipart-exercise-card', {
+      "deleted-homework": @props.task.type is 'homework' and @props.task.is_deleted
+    })
 
     if @isSinglePart()
       @renderSinglePart()
     else
       footer ?= @renderFooter() if pinned
-      <CardBody footer={footer} pinned={pinned} className='openstax-multipart-exercise-card'>
+      <CardBody footer={footer} pinned={pinned} className={classes}>
```

Comment:
should this and the preceding `classes` logic be in `Exercise` component on line 154 as well?